server/src/models/Blog/Collection.model.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable consistent-return */
+const mongoose = require('mongoose')
+
+const { Schema } = mongoose
+
+const CollectionSchema = new Schema({
+  name: {
+    type: String,
+    required: true,
+    index: { unique: true },
+    lowercase: true
+  },
+  created_at: {
+    type: Date,
+    default: Date.now
+  },
+  updated_at: {
+    type: Date,
+    default: Date.now
+  }
```

Comment:
I think we should be more consistent with naming as I have used camel case naming like createdAt, updatedAt . Either change the BlogPost schema to created_at and updated_at or vice versa

Reply:
snake_case will be standard in db layer (mongoose shemas)